mock/src/transaction.rs

```diff
@@ -182,7 +182,7 @@ impl MockTransaction {
     }
 
     /// Set chain_id field for the MockBlock.
-    pub fn chain_id(&mut self, chain_id: Word) -> &mut Self {
+    pub(crate) fn chain_id(&mut self, chain_id: Word) -> &mut Self {
```

Comment:
The documentation line mentions `MockBlock` but I think it should be `MockTransaction`.

Reply:
Ohhh My bad!!!

Reply:
It's not just this one!! All of the functions had `MockBlock`!!!!! :alarm_clock: 
Good catch!!